README.md

```diff
@@ -94,22 +94,36 @@ The typed port connections provide strong compile-time guarantees of correctness
 
 ## F´ Release Notes
 
-#### Release 1.0: 
+### Release 2.0: Release and Migration Notes
 
- * This is the initial release of the software to open source. See the license file for terms of use.
+Version 2.0.0 of F´ represents major improvements across the F´ framework. As such, some work may be required to migrate from other versions of F´ to the new 
+functionality. This section will offer recommendations to migrate to version 2.0.0 of F´.
 
-#### Release 1.01
+* F´ tooling (fprime-util and fprime-gds) should be installed using `pip install fprime-tools fprime-gds`
+* `Os::File::open` with the mode CREATE will now properly respect O_EXCL and error if the file exists. Pass in `false` as the final argument to override. 
+* Revise uses of `Fw::Buffer` to correct usage of member functions using camel case.  E.g. `Fw::Buffer::getsize` is now `Fw::Buffer::getSize`
+* The ground interface chain has been refactored. Projects may switch to using `Svc::Framer`, `Svc::Deframer`, and any implementor of `Drv::ByteStreamDriverModel` to supply the data.  See: TBD.  To continue using the old interface with the GDS run `fprime-gds --comm-checksum-type fixed`.
+* `Svc::FileDownlink` may now be configured to turn off certain errors
+* `Svc::BufferManager` has been reworked to remove errors. When instantiating it please supply a memory allocator. `Ref` has an example of this.
+* A project must now enable F´ framework UTs using `fprime-util generate --ut -DFPRIME_ENABLE_FRAMEWORK_UTS=ON`.  Project defined UTs are unaffected.
+* Dictionaries, binaries, and other build output now are written to a deployments `build_artifacts` folder.
```

Comment:
Do we want to add a note about the removal of the make based build system?

Do we want to add notes about features we added to this release?

Reply:
Are there features missing?  I thought I added most of the "big" features.  I identified them by crawling all the PRs since 1.5.4.

Reply:
A minor missed issue is adding a filedownlink port and queue. But the notes discuss the modifications in terms of needed migrations, but I feel that kinda downplays the improvements made. We should advertise and be proud of the new ground chain and re-written IP drivers!

Reply:
Ok I addressed this concerns